frontend/static/js/view_volume.js

```diff
@@ -149,6 +150,9 @@ function fillPage(data, api_key) {
 	description.innerHTML = data.description;
 	mobile_description.innerHTML = data.description;
 
+	//Issues as Volumes
```

Comment:
Very picky but should be `// Issues as volumes`. Space after slash and lower `v`.